Dockerfile

```diff
@@ -40,6 +39,13 @@ RUN mkdir -p $RAILS_ROOT/tmp/pids
 # Set our working directory inside the image
 WORKDIR $RAILS_ROOT
 
+# Install Node
+COPY .node-version ./
+ENV PATH=/usr/local/node/bin:$PATH
```

Comment:
Hi @javierm, I think we need to add the folder (`/usr/local/node/bin`), where we install NodeJS, to the `/etc/sudoers.d/secure_path` in line 30,  so the node binaries are available for the consul user. Without this change, I have the Javascript Runtime not found error when running any command that loads the application.

Reply:
@Senen Awesome! :tada: I've just updated the pull request.